### ISSUE TITLE
please help me

### ISSUE THREAD

**Issue by mywish1** [algoprog/rivr-search#3]
currently i m not understanding this part

Edit install/sphinx.conf.in (mysql_all source) with your MySQL host, username, password and database name and change all the file paths to yours.

Use install/sphinx.conf.in to create the Sphinx indexes:

    sudo -u sphinx indexer --config sphinx.conf.in --all

Access Sphinx indexes with MySQL client:

    mysql -h 0 -P 9306

Execute the following commands to attach regular indexes to real-time indexes:

    ATTACH INDEX orig TO RTINDEX rtindex;

    ATTACH INDEX mysql0 TO RTINDEX rtindex0;

    ATTACH INDEX mysql1 TO RTINDEX rtindex1;

    ATTACH INDEX mysql2 TO RTINDEX rtindex2;

    ATTACH INDEX mysql3 TO RTINDEX rtindex3;

Start the Sphinx search daemon.

can you tel me how to fix this issue i have updated database details in install/spinux.confg.in file now what i do with that file should i need to upload this in somewhere and also sphinx search engine is not installed in my server currently should i need to install them in my server if yes then plz tel me method how to install them thanks waiting

**Comment by algoprog** [comment]
Obviously you should download and install Sphinx first from http://sphinxsearch.com/downloads/release/

**Comment by mywish1** [comment]
yes i did install and now its ok now what is next its install /etc/sphinx/sphinx.config now should i need to replace with sphinx/config.in file with them and what is path i need to set there plz update thanks.

**Comment by algoprog** [comment]
You should edit the configuration file in install folder and copy it to Sphinx folder. I can't tell you how to install Sphinx. If you have any specific problems I can help you.

**Comment by mywish1** [comment]
where i need to upload sphinx config file i mean sphinx install in etc/sphinx there is already 1 sphinx.conf file there so should i need to edit that one or edit install/sphinx.cong.in file and upload there there if suppose i upload that file there then what is path i need to replace or change them can you share your skype id we can discuss each other with live chat to easily understand thanks.

**Comment by mywish1** [comment]
even i replace install/sphinx.cong.in file with orignal sphinx file and also change database details but still not working i dont know what is going wrong plz give me your live im id or anything where we can chat each other thanks.

**Comment by mywish1** [comment]
my home page showing blank nothing to show up plz tel me how to fix that problem and thanks for that

**Comment by mywish1** [comment]
when i try to run this command got this error
sudo -u sphinx indexer --config sphinx.conf.in --all

take a look screenshot
http://prntscr.com/duzhtn

and when i try to run and excute these commands then got this 
mysql -h 0 -P 9306

ATTACH INDEX orig TO RTINDEX rtindex;

ATTACH INDEX mysql0 TO RTINDEX rtindex0;

take a look screenshot
http://prntscr.com/duzijz

**Comment by arvs47** [comment]
hello can i setup up it into a linux server, i cant find any article to install sphinx on linux web server ,did you setup this in a local pc  because having problem with sphinx..
 why the home page is blank ,, if you have time please help

**Comment by raajastra** [comment]
Hi algoprog,

can you install this script on my webserver and get it work? I will pay thru donation.
please help..

thanks
